app/models/search_builder.rb

```diff
@@ -96,7 +96,8 @@ def process_begins_with(value, op)
   end
 
   def process_is(value, op)
-    return value if value.match(/"/) rescue true
+    return if value.blank?
```

Comment:
Arg it sees not to want to return nil if value is ""...

```suggestion
    return if value.nil?
```

I'm really confused by this error.. it's not happening at this method just farther down the stack (which is part of my confusion)